.github/workflows/ci.yaml

```diff
@@ -108,12 +110,21 @@ jobs:
           RUSTPYTHONPATH: ${{ github.workspace }}/Lib
         if: runner.os == 'Linux'
       - name: run cpython tests (macOS lightweight)
-        run: target/release/rustpython -m test -x test_argparse -x test_json -x test_bytes -x test_long -v
+        run:
+          target/release/rustpython -m test -v -x
+            test_argparse test_json test_bytes test_bytearray test_long test_unicode test_array
+            test_asyncgen test_list test_complex test_json test_set test_dis test_calendar
+            test_importlib
```

Comment:
importlib depends on filesystem. I think this is better to be not skipped.

Reply:
Should I unskip it for windows as well?

Reply:
I think so if possible. The winodws tests with `-x` were failing at the time I added it.